content/apidocs-mxsdk/apidocs/team-server-api.md

```diff
@@ -4,203 +4,411 @@ category: "API Documentation"
 
-##### 3.1.2.2 Example
+##### 4.1.3.1 Payload Example
 
 ```json
-[{
-     "Name" :  "trunk" ,
-     "LatestRevisionNumber" :  9 ,
-     "LatestRevisionMendixVersion" :  "5.6.0" ,
-     "DisplayName" :  "Main line"
-},{
-     "Name" :  "statistical functions" ,
-     "LatestRevisionNumber" :  13 ,
-     "LatestRevisionMendixVersion" :  "5.6.0" ,
-     "DisplayName" :  "statistical functions"
-}]
+{
+  "projectId": "c0af1725-edae-4345-aea7-2f94f7760e33",
+  "type": "svn",
+  "url": "https://teamserver.sprintr.com/c0af1725-edae-4345-aea7-2f94f7760e33/",
+  "isMendixHosted": true,
+  "defaultBranchName": "trunk"
+}
 ```
 
-### 3.2 Retrieve Branch {#retrieve-branch}
+##### 4.1.3.2 Error Codes
+
+| HTTP Status | Title | Detail |
+| --- | --- | --- |
+| 400 | Bad Request | Invalid app ID |
+| 401 | Unauthorized | Invalid token |
+| 403 | Forbidden | Access denied |
+| 404 | Not Found | Repository not found |
+| 500 | Internal Server Error | Something went wrong |
+
+Error Response format and examples are given in section: [Error response format](#3-error-response-format)
+
+### 4.2 Retrieve Branches
 
-Retrieves a specific branch that belongs to the team server project of a specific app which the authenticated user has access to as a regular user.
+Returns information about the branches of the version control repository for a Mendix project.
+
+The response is paginated using cursor-based pagination.
 
 ```http
- HTTP Method: GET
- URL: https://deploy.mendix.com/api/1/apps/<AppId>/branches/<Name>
+HTTP Method: GET
+ URL: https://teamserver.api.mendix.com/v1/repositories/<appId>/branches
 ```
 
-#### 3.2.1 Request
+#### 4.2.1 Request
 
-##### 3.2.1.1 Parameters
+##### 4.2.1.1 Path Parameter
 
-*   _AppId_ (String) : Subdomain name of an app.
-*   _Name_ (String) : Name of the branch to get or 'trunk' to get the main line. The name of the branch should be [URL-encoded](https://www.w3schools.com/tags/ref_urlencode.asp).
+|Name|Type|Required|Description|
+|---|---|---|---|
+|`AppId`|String|Yes|The app ID (also known as project ID) of the Mendix project for which the repository information should be returned.|
 
-##### 3.2.1.2 Example
+##### 4.2.1.2 Query Parameter
 
-```http
-GET /api/1/apps/calc/branches/statistical%20functions HTTP/1.1
-Host: deploy.mendix.com
+|Name|Type|Required|Description
+|---|---|---|---|
+|`limit`|Integer|No|The number of items to return per result page. Defaults to 20 items. Maximum is 100 items.|
+|`cursor`|String|No|A cursor specifying which page to retrieve. To obtain a cursor value, see the `cursors` property of the response payload of this operation. If no cursor is specified, the first page is returned. Note: To the user of this API, a cursor is an opaque value that can only be obtained from a previous API response.|
```

Comment:
Use the note style:
{{% alert type="note" %}}
To the user of this API, a cursor is an opaque value that can only be obtained from a previous API response.
{{% /alert %}}

Reply:
Style applied.